src/components/app/MenuButtons/index.css

```diff
@@ -45,10 +17,25 @@
 }
 
 .menuButtonsButton:hover {
-  background-image: linear-gradient(rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1));
```

Comment:
I believe we had a `background-image` because previously we had some real gradients in some cases. But that's legacy now.